.dockerignore

```
@@ -74,6 +74,8 @@ cpu.out
 /VERSION
 /.air
 /.go-licenses
+Dockerfile
+Dockerfile.rootless
```

```suggestion
/Dockerfile
/Dockerfile.rootless
```

In any case, let's match exactly.